tests/conftest.py

```diff
@@ -174,98 +216,172 @@ def standard_image_key_path():
 def diad_loader():
     return "tests/samples/loader_configs/diad.yaml"
 
 
 @pytest.fixture
-def diad_pipeline_gpu():
-    return "tests/samples/pipeline_template_examples/DLS/01_diad_pipeline_gpu.yaml"
+def standard_loader():
+    return "tests/samples/loader_configs/standard_tomo.yaml"
 
 
 @pytest.fixture
-def i12_data():
-    return "tests/test_data/i12/separate_flats_darks/i12_dynamic_start_stop180.nxs"
+def sample_pipelines():
+    return "tests/samples/pipeline_template_examples/"
+
+
+###########Auto-generated pipelines##################
 
 
 @pytest.fixture
-def pipeline360():
-    return "samples/pipeline_template_examples/DLS/02_i12_360scan_pipeline.yaml"
+def cpu_pipeline_gridrec():
+    return "docs/source/pipelines_full/cpu_pipeline_gridrec.yaml"
 
 
 @pytest.fixture
-def i12_loader():
-    return (
-        "tests/samples/pipeline_template_examples/DLS/03_i12_separate_darks_flats.yaml"
-    )
+def gpu_pipelineFBP():
+    return "docs/source/pipelines_full/gpu_pipelineFBP.yaml"
 
 
 @pytest.fixture
-def i12_loader_ignore_darks_flats():
-    return "tests/samples/pipeline_template_examples/DLS/04_i12_ignore_darks_flats.yaml"
+def gpu_pipelineFBP_denoising():
+    return "docs/source/pipelines_full/gpu_pipelineFBP_denoising.yaml"
 
 
 @pytest.fixture
-def standard_loader():
-    return "tests/samples/loader_configs/standard_tomo.yaml"
+def gpu_pipeline_diad_FBP_noimagesaving():
+    return "docs/source/pipelines_full/gpu_diad_FBP_noimagesaving.yaml"
 
 
 @pytest.fixture
-def sample_pipelines():
-    return "tests/samples/pipeline_template_examples/"
+def gpu_pipeline_diad_FBP():
+    return "docs/source/pipelines_full/gpu_diad_FBP.yaml"
+
+
+@pytest.fixture
+def gpu_pipeline_360_paganin_FBP():
+    return "docs/source/pipelines_full/gpu_360_paganin_FBP.yaml"
+
+
+# ---------------------END------------------------#
+
+###########Raw projection data (big)##################
+# Note that raw_data folder should exist with the datasets
+# listed bellow
+
+
+@pytest.fixture
+def diad_k11_38727():
+    # 4k projections, 45gb dataset
+    return "tests/test_data/raw_data/diad/k11-38727.nxs"
+
+
+@pytest.fixture
+def diad_k11_38729():
+    # 2k projections, 22gb dataset
+    return "tests/test_data/raw_data/diad/k11-38729.nxs"
+
+
+@pytest.fixture
+def diad_k11_38730():
+    # 1k projections, 11gb dataset
+    return "tests/test_data/raw_data/diad/k11-38730.nxs"
+
+
+@pytest.fixture
+def diad_k11_38731():
+    # 0.5k projections, 6gb dataset
+    return "tests/test_data/raw_data/diad/k11-38731.nxs"
+
+
+@pytest.fixture
+def i13_177906():
+    # 2.5k projections, 27gb dataset
+    return "tests/test_data/raw_data/i13/177906.nxs"
+
+
+@pytest.fixture
+def i13_179623():
+    # 6k projections, 65gb dataset, 360 degrees scan
+    return "tests/test_data/raw_data/i13/360/179623.nxs"
+
+
+# @pytest.fixture
+# def i13_180622():
+#     # 6k projections, 65gb dataset, 360 degrees scan
+#     return "tests/test_data/raw_data/i13/360/180622.nxs"
```

Comment:
Any reason why this is commented-out? Is it old and should be removed? Does the pipeline not run for some reason (in which case, maybe linking to a github issue or something would be helpful)?

Reply:
Yeah, I mentioned that above. The data is a bit too complex so that stitching methods do not work well. We probably need a test for something more representative, so I added 179623 instead.